ledger/participant-state/kvutils/tools/BUILD.bazel

```diff
@@ -5,13 +5,37 @@ load(
     "//bazel_tools:scala.bzl",
     "da_scala_benchmark_jmh",
     "da_scala_binary",
+    "da_scala_library",
     "lf_scalacopts",
 )
 
+da_scala_library(
+    name = "tools",
+    srcs = glob(["src/main/scala/**/*.scala"]),
+    tags = ["maven_coordinates=com.daml:data-export-kvutils:__VERSION__"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//daml-lf/data",
+        "//daml-lf/engine",
+        "//ledger/ledger-on-memory",
+        "//ledger/metrics",
+        "//ledger/participant-state",
+        "//ledger/participant-state/kvutils",
+        "//ledger/participant-state/kvutils:daml_kvutils_java_proto",
+        "//libs-scala/direct-execution-context",
+        "@maven//:com_google_protobuf_protobuf_java",
+        "@maven//:com_typesafe_akka_akka_actor_2_12",
+        "@maven//:com_typesafe_akka_akka_stream_2_12",
+        "@maven//:io_dropwizard_metrics_metrics_core",
+    ],
+)
+
 da_scala_binary(
     name = "integrity-check",
-    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/IntegrityCheck.scala"],
-    main_class = "com.daml.ledger.participant.state.kvutils.tools.IntegrityCheck",
+    srcs = ["src/main/scala/com/daml/ledger/participant/state/kvutils/tools/driver/IntegrityCheck.scala"],
+    main_class = "com.daml.ledger.participant.state.kvutils.tools.driver.IntegrityCheck",
```

Comment:
Why do we need to keep V1?

Reply:
For backwards compatibility reasons. It'll make sense to phase out v1 once we start publishing ledger exports in the new format and all build tools will be using v2.